internal/controller/api/v1beta1/agent_types.go

```diff
@@ -24,45 +24,45 @@ import (
-	AgentReadyReason                  string                     = "AgentIsReady"
-	AgentReadyMsg                     string                     = "The agent is ready to begin the installation"
-	AgentNotReadyReason               string                     = "AgentNotReady"
-	AgentNotReadyMsg                  string                     = "The agent is not ready to begin the installation"
-	AgentAlreadyInstallingReason      string                     = "AgentAlreadyInstalling"
-	AgentAlreadyInstallingMsg         string                     = "The agent cannot begin the installation because it has already started"
-	UnknownStatusReason               string                     = "UnknownStatus"
-	UnknownStatusMsg                  string                     = "The installation status is currently not recognized:"
-	ValidatedCondition                conditionsv1.ConditionType = "Validated"
-	AgentValidationsPassingReason     string                     = "ValidationsPassing"
-	AgentValidationsPassingMsg        string                     = "The agent's validations are passing"
-	AgentValidationsUnknownReason     string                     = "ValidationsUnknown"
-	AgentValidationsUnknownMsg        string                     = "The agent's validations have not yet been calculated"
-	AgentValidationsFailingReason     string                     = "ValidationsFailing"
-	AgentValidationsFailingMsg        string                     = "The agent's validations are failing:"
+	SpecSyncedCondition           conditionsv1.ConditionType = "SpecSynced"
+	SyncedOkReason                string                     = "SyncOK"
+	SyncedOkMsg                   string                     = "The Spec has been successfully applied"
+	BackendErrorReason            string                     = "BackendError"
+	BackendErrorMsg               string                     = "The Spec could not be synced due to backend error:"
+	InputErrorReason              string                     = "InputError"
+	InputErrorMsg                 string                     = "The Spec could not be synced due to an input error:"
+	NotAvailableReason            string                     = "NotAvailable"
+	NotAvailableMsg               string                     = "Information not available"
+	ConnectedCondition            conditionsv1.ConditionType = "Connected"
+	AgentConnectedReason          string                     = "AgentIsConnected"
+	AgentDisconnectedReason       string                     = "AgentIsDisconnected"
+	AgentConnectedMsg             string                     = "The agent's connection to the installation service is unimpaired"
+	AgentDisonnectedMsg           string                     = "The agent has not contacted the installation service in some time, user action should be taken"
+	InstalledCondition            conditionsv1.ConditionType = "Installed"
+	InstalledReason               string                     = "InstallationCompleted"
```

Comment:
I'm not sure that it belongs to this PR. if you are adding it anyway then please update the commit message.

Reply:
I renamed these to more general to be able to reuse in ClusterDeployment conditions as the text is the same.
Do you think I need to duplicate them and have it in clusterdeployments_controller.go?

Reply:
I don't think that we will split the controllers into different services, but if they are common then maybe they belong to a different file?

Reply:
I could move all the conditions constants to internal/controller/controllers/common.go.
WDYT?

Reply:
Sure sounds good to me.

Reply:
moved to conditions.go